### PR TITLE
GSYE-650: Exposed min offer / bid age to the constructor arguments of…

### PR DESCRIPTION
**Pull request by spyrostz**
… the Area class.

## Reason for the proposed changes

Please describe what we want to achieve and why.

## Proposed changes

-

**INTEGRATION_TESTS_BRANCH**=master
**GSY_FRAMEWORK_BRANCH**=feature/GSYE-650-heatpump-fixes

### CODE REVIEW

**Review by hannesdiedrich — APPROVED** [review]
In understand that this is for creating a separate heat market, right? 
LGTM

**Review by hannesdiedrich — APPROVED** [review]
LGTM

File: src/gsy_e/models/strategy/state/heat_pump_state.py [hunk 1]
```diff
@@ -143,14 +152,16 @@ def delete_past_state_values(self, current_time_slot: DateTime):
...
     def get_results_dict(self, current_time_slot: DateTime) -> Dict:
-        return {
+        retval = {
             "storage_temp_C": self.get_storage_temp_C(current_time_slot),
             "temp_decrease_K": self.get_temp_decrease_K(current_time_slot),
             "temp_increase_K": self.get_temp_increase_K(current_time_slot),
             "energy_consumption_kWh": self.get_energy_consumption_kWh(current_time_slot),
             "max_energy_demand_kWh": self.get_max_energy_demand_kWh(current_time_slot),
             "min_energy_demand_kWh": self.get_min_energy_demand_kWh(current_time_slot),
+            "total_traded_energy_kWh": self._total_traded_energy_kWh[current_time_slot],
```
Comment (hannesdiedrich) [inline src/gsy_e/models/strategy/state/heat_pump_state.py:162]: minor, but it would be consistent to also add a getter method for this

Comment (spyrostz) [reply src/gsy_e/models/strategy/state/heat_pump_state.py:211]: As discussed offline, the total_traded_energy is not used outside of the scope of this class, therefore a potential `get_total_traded_energy_kWh` would not be used by another part of the code. We can add it once we need it.

**Review by hannesdiedrich — APPROVED** [review]
LGTM

**Pull request merged**